tour_model_eval/load_predict.py

```diff
@@ -0,0 +1,92 @@
+import emission.core.get_database as edb
+import emission.analysis.modelling.tour_model.similarity as similarity
+import emission.analysis.modelling.tour_model.data_preprocessing as preprocess
+import jsonpickle as jpickle
+
+
+def loadModelStage(filename):
+    import emission.analysis.classification.inference.mode.seed.pipeline as seedp
+    model = loadModel(filename)
+    return model
```

Comment:
why do you need this import?
In general, you should not use anything from the mode inference pipeline, since that has separate functionality. It is just an example.

Reply:
If I remove this, there will be an error.
```
AttributeError: attribute 'dtype' of 'numpy.generic' objects is not writable
```
But I figure that you have these two line in your pipeline file. So I put them in my function. It works now.
```
    import jsonpickle.ext.numpy as jsonpickle_numpy
    jsonpickle_numpy.register_handlers()
```

Reply:
is there a new change showing this so I can resolve this conversation?